tests/testthat/test-mwrite.R

```diff
@@ -230,3 +230,43 @@ test_that("code gets appropriately quoted", {
   lines <- readLines(cpp)
   expect_match(lines, 'outvars = "A"', all=FALSE)
 })
+
+test_that("model with lower case names", {
```

Comment:
missing `skip_if_not_installed("yaml")`

new test looks good to me otherwise

Reply:
Thanks.